### DISCUSSION TITLE
充值页面打不开

### DISCUSSION THREAD

**Discussion by xrz-baizer** [ZuodaoTech/everyone-can-use-english#722] [General]
就一直在这个页面
<img width="800" alt="image" src="https://github.com/xiaolai/everyone-can-use-english/assets/48161253/ea383d6d-a91a-4193-b3b7-818a92aa4d80">

**Comment by an-lee** [comment]
应该是因为你的网络连不了外网

**Reply by xrz-baizer** [reply]
OK，开了全局代理就好了。 规则代理模式进不去。
